ci-operator/step-registry/openshift/e2e/aws/localzone/byo-vpc/openshift-e2e-aws-ovn-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: openshift-e2e-aws-localzone-byo-vpc
+  steps:
+    pre:
+    - chain: ipi-aws-pre-localzone
+    test:
+    - ref: openshift-e2e-test
```

Comment:
we may need to add a custom LZ tests here, for example:
- check if LZ node is running with expected labels
- check if MTU is set correctly
- create an app on LZ, using the node selectors for it (make sure the image uses the internal registry)
- quickly measure the latency between zones in the region and LZ by curl-ing from nodes in the region

Reply:
more coverage will be implemented on https://issues.redhat.com/browse/SPLAT-1028